sdk/openai/Azure.AI.OpenAI/tests/CompletionsTests.cs

```diff
@@ -133,15 +132,48 @@ public async Task AdvancedCompletionsOptions(Service serviceTarget)
 
             string choiceText = choice.Text;
             Assert.That(choiceText, Is.Not.Null.Or.Empty);
-            Assert.That(choiceText.Length, Is.GreaterThan(promptText.Length));
-            Assert.That(choiceText.ToLower().StartsWith(promptText.ToLower()));
-            Assert.That(choiceText.Substring(promptText.Length).Contains(" banana"), Is.False);
+            Assert.That(choiceText.ToLower().StartsWith(promptText.ToLower()), Is.False);
 
             Assert.That(choice.LogProbabilityModel, Is.Not.Null.Or.Empty);
             Assert.That(choice.LogProbabilityModel.Tokens, Is.Not.Null.Or.Empty);
             Assert.That(response.Value.Usage.TotalTokens, Is.GreaterThan(response.Value.Choices[0].LogProbabilityModel.Tokens.Count));
         }
 
+        [RecordedTest]
+        [TestCase(Service.Azure)]
+        [TestCase(Service.NonAzure)]
+        public async Task AdvancedCompletionsOptionsWithEcho(Service serviceTarget)
+        {
+            OpenAIClient client = GetTestClient(serviceTarget);
+            string deploymentOrModelName = GetDeploymentOrModelName(serviceTarget, Scenario.LegacyCompletions);
+            string promptText = "Are bananas especially radioactive?";
+            var requestOptions = new CompletionsOptions()
+            {
+                DeploymentName = deploymentOrModelName,
+                Prompts = { promptText },
+                Temperature = 0.75f,
+                User = "AzureSDKOpenAITests",
+                Echo = true,
```

Comment:
oh, I should have read a bit further -- it looks like it was just expanded and split. If that's the case, looks like it's all taken care of! (I'll leave active just to confirm, though)

Reply:
Your understanding is correct. We have a seperated test for covering `echo` case.